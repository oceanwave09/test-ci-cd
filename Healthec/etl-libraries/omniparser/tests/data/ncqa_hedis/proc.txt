95057           2021112811075005            S20211128EVN                    
95060           1953121190870               C19531211EVN                    
95063           2021071290870               C20210712EVN                    
95111           202107051010696002          S20210705INT                    
95218           20220302391054008           S20220302INT                    