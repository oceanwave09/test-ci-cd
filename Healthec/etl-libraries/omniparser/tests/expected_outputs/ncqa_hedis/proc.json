[
	{
		"code_flag": "S",
		"end_date": "20211128",
		"member_id": "95057",
		"procedure_code": "11075005",
		"row_id": "2eaa1966-8c9d-4365-b44a-3359d44c3db8",
		"service_date": "20211128",
		"service_status": "EVN"
	},
	{
		"code_flag": "C",
		"end_date": "19531211",
		"member_id": "95060",
		"procedure_code": "90870",
		"row_id": "a192c85f-ba98-4468-84c5-e4e1f13a34eb",
		"service_date": "19531211",
		"service_status": "EVN"
	},
	{
		"code_flag": "C",
		"end_date": "20210712",
		"member_id": "95063",
		"procedure_code": "90870",
		"row_id": "6a299942-8fff-45bb-9a8c-62fe5ba1a953",
		"service_date": "20210712",
		"service_status": "EVN"
	},
	{
		"code_flag": "S",
		"end_date": "20210705",
		"member_id": "95111",
		"procedure_code": "1010696002",
		"row_id": "213dd3ba-10ab-4d0b-a796-000e98de790d",
		"service_date": "20210705",
		"service_status": "INT"
	},
	{
		"code_flag": "S",
		"end_date": "20220302",
		"member_id": "95218",
		"procedure_code": "391054008",
		"row_id": "e9354c04-f7d4-4c25-93a7-1069e99687fc",
		"service_date": "20220302",
		"service_status": "INT"
	}
]
